src/machines/getPIDCredentialMachine.tsx

```diff
@@ -218,32 +221,63 @@ const createGetPIDCredentialMachine = (opts?: CreateGetPIDCredentialsMachineOpts
+                },
```

Comment:
jezus, that was quite a scroll-down. Can't you put this in a json file? Something like
```
{
  "pid": {
    "backgroundImage": "data:image/png;base64,...",
    "logo": "data:image/png;base64,..."
  }
}
```

The in this file
```
import pidAssets from '....../pid-assets.json';
...
  background: {
    image: {
      uri: pidAssets .pid.backgroundImage
    }
  },
  logo: pidAssets.pid.logo
```

Or at least as constant in a separate branding file.

Reply:
fixed